docs/migration_guide.md

```diff
@@ -14,61 +14,83 @@ Since the migration might alter the database schema and the settings of `harbor.
 
-    You can pull the image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
     ```sh
     docker pull goharbor/harbor-migrator:[tag]
     ```
 
-    Alternatively, if you are using an offline installer package you can load it from the image tarball included in the offline installer package. Replace [version] with the release version of Harbor (e.g. v1.5.0) in the below command:
+    **Offline installer** can load it from the image tarball included in the offline installer package. Replace [version] with the release version of Harbor (e.g. v1.5.0) in the below command:
+
     ```sh
     tar zxf <offline package>
     docker image load -i harbor/harbor.[version].tar.gz
     ```
 
-5. If you are current version is v1.7.x or earlier, i.e. migrate config file from `harbor.cfg` to `harbor.yml`.
+2. **If current Harbor version is v1.7.x or earlier**, You need migrate config file from `harbor.cfg` to `harbor.yml`. So need mount both `harbor.cfg` and `harbor.yml` to run the command.
 
-    **NOTE:** You can find the ${harbor_yml} in the extracted installer you got in step `3`, after the migration the file `harbor.yml`
-    in that path will be updated with the values from ${harbor_cfg}
+    **NOTE:** After the migration the `harbor.yml` file will be updated with the values from `${harbor_cfg}`
 
     ```sh
     docker run -it --rm -v ${harbor_cfg}:/harbor-migration/harbor-cfg/harbor.yml -v ${harbor_yml}:/harbor-migration/harbor-cfg-out/harbor.yml goharbor/harbor-migrator:[tag] --cfg up
     ```
 
-    Otherwise, If your version is 1.8.x or higher, just upgrade the `harbor.yml` file.
+    **If current Harbor version is v1.8.0 or above**, there is no `harbor.cfg` file anymore. So, only `harbor.yml` file need to mount.
+
+    **NOTE:** This command only update the file ` ${harbor_yml}`. You should copy this file to the correct place by yourself after upgrade.
 
     ```sh
     docker run -it --rm -v ${harbor_yml}:/harbor-migration/harbor-cfg/harbor.yml goharbor/harbor-migrator:[tag] --cfg up
     ```
 
     **NOTE:** The schema upgrade and data migration of the database is performed by core when Harbor starts, if the migration fails, please check the log of core to debug.
 
-6. Under the directory `./harbor`, run the `./install.sh` script to install the new Harbor instance. If you choose to install Harbor with components such as Notary, Clair, and chartmuseum, refer to [Installation & Configuration Guide](../docs/installation_guide.md) for more information.
+    
+
+#### Install Harbor
+
+Under the directory `./harbor`, run the `./install.sh` script to install the new Harbor instance. If you choose to install Harbor with components such as Notary, Clair, and chartmuseum, refer to [Installation & Configuration Guide](../docs/installation_guide.md) for more information.
```

Comment:
Run `./install.sh` in the `./harbor` directory to install the new Harbor instance. For instructions about how to install Harbor with components such as Notary, Clair, and chartmuseum, see the [Installation & Configuration Guide](../docs/installation_guide.md).